index.html

```diff
@@ -32,13 +38,194 @@ <h2>I'm a Software Developer</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+        <div class="project">
+          <div class="head_card">
+            <h2>Professional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about_me">
+      <div class="about">
+        <h1>About me</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button>Get My Resume</button>
+      </div>
+      <div class="frame">
+        <img src="./assets/images/Frame1.png" alt="" />
+      </div>
```

Comment:
- [ ] Good job so far your website is awesome! 👏 , since the background image of the about me section is a background, it should be implemented as a CSS background-image and not a separate `<img>` tag,kindly fix that 🙏 

![image](https://user-images.githubusercontent.com/89282221/198714959-f916f517-222a-465e-a26b-5b09ade506d5.png)